hack/common.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# The root of the build/dist directory
+readonly II_ROOT=$(
+  unset CDPATH
+  ii_root=$(dirname "${BASH_SOURCE}")/..
+
+  cd "${ii_root}"
+  ii_root=`pwd`
+  if [ -h "${ii_root}" ]; then
+    readlink "${ii_root}"
+  else
+    pwd
+  fi
+)
+
+readonly II_GOPATH=$(
+  unset CDPATH
+  cd ${II_ROOT}/../../../..
+  pwd
+)
+
+readonly II_GO_PACKAGE=github.com/simon3z/image-inspector
```

Comment:
actually, it looks like your repo is upstream for openshift?  Just let me know if this is wrong.  Basically this script is expecting to find the path structure to look like whatever that upstream is just like we require in OpenShift